### ISSUE TITLE
BPMN files with multiple diagrams

### ISSUE THREAD

**Issue by falko** [camunda/camunda-modeler#524]
The [upcoming demo of the OMG's BPMN Model Interchange Working Group (MIWG)](https://github.com/bpmn-miwg/bpmn-miwg-demos/tree/master/2017-06-omg-technical-meeting-brussels) will showcase BPMN files containing multiple diagrams. Is there a chance to support this in the modeler?

Here is the current example file:
[Multi-diagram.bpmn.zip](https://github.com/camunda/camunda-modeler/files/761337/Multi-diagram.bpmn.zip)

It contains the following three diagrams:
![it](https://cloud.githubusercontent.com/assets/40038/22748016/0261fa56-ee29-11e6-99f8-b1f76db35265.png)
![facilities](https://cloud.githubusercontent.com/assets/40038/22748026/069907ea-ee29-11e6-8ba7-9430598c3878.png)
![payroll](https://cloud.githubusercontent.com/assets/40038/22748027/06ce9cca-ee29-11e6-827c-c1d95ae54781.png)

Only the 'IT' diagram shows up in Camunda Modeler. There are no XML errors reported in the log.

**Comment by nikku** [comment]
How should we support multiple diagrams? Is switching between the diagrams a suitable way to go?

**Comment by falko** [comment]
Yes, totally. Other vendors show them on multiple tabs at the bottom, just like we already show the switch between diagram and XML.

**Comment by falko** [comment]
A BPMN file with multiple diagrams has become an official test case of the BPMN MWIG Test Suite:
https://github.com/bpmn-miwg/bpmn-miwg-test-suite/blob/master/Reference/C.4.0.bpmn

**Comment by nikku** [comment]
This needs to be addressed via https://github.com/bpmn-io/bpmn-js/issues/87.

**Comment by nikku** [comment]
Infrastructure provided upstream. We need to decide if we'd like to support this in the modeler visually, too.

**Comment by barmac** [comment]
There is a plugin which implements this feature request: https://github.com/sharedchains/camunda-modeler-plugin-multidiagram/
